### DISCUSSION TITLE
Kibana web not open

### DISCUSSION THREAD

**Discussion by Aabayoumy** [sherifabdlnaby/elastdocker#103] [Questions & Troubleshooting]
on web :

Kibana server is not ready yet.

i tried some solutions from https://stackoverflow.com/questions/58011088/kibana-server-is-not-ready-yet
but not work for me 
thx

**Comment by Aabayoumy** [comment]
kibana container log [2023-06-03T07:57:05.668+00:00][ERROR][elasticsearch-service] Unable to retrieve version information from Elasticsearch nodes. connect ECONNREFUSED 172.25.0.3:9200

**Comment by mostafahussein** [comment]
Hello Ahmed,
Have you checked elasticseatch logs?

It says 9200 connection refused, which means that elasticsearch is not running, you need to check its logs to see the actual issue.
